pkg/storage/chunk/azure/blob_storage_client.go

```diff
@@ -259,7 +305,7 @@ func (b *BlobStorage) List(ctx context.Context, prefix, delimiter string) ([]chu
 }
 
 func (b *BlobStorage) DeleteObject(ctx context.Context, blobID string) error {
-	blockBlobURL, err := b.getBlobURL(blobID)
+	blockBlobURL, err := b.getBlobURL(blobID, false)
```

Comment:
Isn't this operation safe? It's only the line `313` that should not be hedged, right?

Reply:
Not sure what you means, only gets are hedged.

Reply:
Ah sorry, I see `blockBlobURL` is used by the `Delete` call; I thought this was a separate `GET` operation

Reply:
see above.